app/main.py

```
@@ -155,6 +172,13 @@
         [chat_id, user_id, chat_id]
     )
     redis.set(f"{user_id}-{chat_id}.world", json.dumps(["fantasy", "high magic"]))
+    try:
+        sql_connection.cursor().execute(
+            "INSERT INTO `chat_users`.`statistics` (label, value) VALUES (?, 1) ON DUPLICATE KEY UPDATE value = value +1;",
+            ['Chats']
+        )
+    except mariadb.Error as error:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(mariadb.Error)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(mariadb.Error)`

(SIM105)